### PR TITLE
Setup to publish one package

### PR DESCRIPTION
**Pull request by igloosi**
This work is the minimum to take the current folder structure into a place where it can be published as one package.

URL: https://govuk-frontend-review-pr-680.herokuapp.com/

Checklist:

- [x] all files are moved into into root
- [x] Main folder is packages instead of package
- [x] Components do not have a package.json
- [x] Lerna is removed
- [x] Can be published as `@govuk-frontend/govuk-frontend`
- [x] Any documentation that refers to this is updated. 
    Note: we have not updated the main readme to avoid providing  #instructions that are not yet correct(before this gets shipped)
- [x] Update any tests/ build pipelines to support this
- [x] Check build:package and build:dist succeeds
- [x] updates release scripts

To test run:
`npm run build:packages`
`npm run build:dist`

We have tested the release process and consumed the package in the Design System.

Trello ticket: https://trello.com/c/AXKG0aAJ/932-5-setup-to-publish-one-package

### CODE REVIEW

File: package/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ To consume all components you must be running npm version 5 or above.
 
 ## Installation
 
-    npm install --save @govuk-frontend/all
+    npm install --save @govuk-frontend/govuk-frontend
```
Comment (nickcolley) [inline package/README.md]: should this say `@govuk-frontend/govuk-frontend-placeholder`?

Comment (igloosi) [reply package/README.md]: it can say `magic-land-awaits`

Comment (igloosi) [reply package/README.md]: yeah it should.

**Comment by nickcolley** [comment]
For other reviewer's context:

This pull request leaves us in a place where you consume the project like so:

```scss
@import 'node_modules/@govuk-frontend/govuk-frontend-placeholder/all`
```

```scss
@import 'node_modules/@govuk-frontend/govuk-frontend-placeholder/button/button`
```

**Comment by nickcolley** [comment]
Looks like the main readme still refers to 'packages'

File: app/assets/scss/app-old-ie.scss [hunk 2]
```diff
@@ -2,5 +2,5 @@
 // right corner of your site during development, add the breakpoints
 // to this list, ordered by width, e.g. (mobile, tablet, desktop).
 $mq-show-breakpoints: (desktop);
-@import "../../../src/all/all-old-ie";
+@import "../../../src/all";
```
Comment (nickcolley) [inline app/assets/scss/app-old-ie.scss]: all-ie8

**Comment by nickcolley** [comment]
When running `build:package` there are folders for snapshots, but not the snapshot files. Not sure if this is an introduced change.

**Comment by hannalaakso** [comment]
Reviewed this with @nickcolley, looks good to me too 🎉

**Comment by igloosi** [comment]
rebased with ALL THE THINGS

**Pull request merged**